src/lib/lspace/elements/root_element.rs

```diff
@@ -124,3 +120,67 @@ impl TContainerElement for RootElement {
     }
 }
 
+impl TBinElement for RootElement {
+    fn get_child(&self) -> Option<Ref<ElementRef>> {
+        let mm = self.m.borrow();
```

Comment:
Might this be clearer as `assert(mm.children.len() < 2)` and then `if mm.children.len() == 0 { None } else {Some(...)}`? I'm not sure though!